build/index.js

```diff
@@ -1,30 +1,37 @@
-const AdmZip = require("adm-zip");
 const fs = require("fs");
 const path = require("path");
 const tmpPath = path.resolve(`${__dirname}/tmp`);
-const elmPath = path.resolve(`${__dirname}/../src/ZondIcons/`);
+const elmPath = path.resolve(`${__dirname}/../src/`);
 const elmJsonFile = path.resolve(`${__dirname}/../elm.json`);
+const elmModuleName = "ZondIcons";
 
 const execShell = (command) => {
   return require("child_process").execSync(command);
 };
 
 const generateElmJson = () => {
-  execShell("yes | npx elm init");
-
-  const json = require(elmJsonFile);
-  json.type = "package";
-  delete json["source-directories"];
-  json["elm-version"] = "0.19.0 <= v < 0.20.0";
-  json.summary = "Zondicons Library";
-  json.license = "BSD-3-Clause";
-  json.name = "pehota/elm-zond-icons";
-  json.version = "1.0.0";
-  json["exposed-modules"] = [];
-  fs.writeFileSync(elmJsonFile, JSON.stringify(json));
+  const json = `
+{
+  "type": "package",
+  "summary": "Zondicons SVG Icons Library",
+  "license": "BSD-3-Clause",
+  "name": "pehota/elm-zond-icons",
+  "version": "1.0.0",
+  "exposed-modules": ["${elmModuleName}"],
+  "elm-version": "0.19.0 <= v < 0.20.0",
+  "dependencies": {
+    "elm/core": "1.0.0 <= v < 2.0.0",
+    "elm/html": "1.0.0 <= v < 2.0.0",
+    "elm/svg": "1.0.0 <= v < 2.0.0"
+  },
+  "test-dependencies": {}
+}
+  `;
+  fs.writeFileSync(elmJsonFile, json);
+  execShell("npx prettier --write elm.json");
```

Comment:
Do you really need that? `json.trim()` should do the job, no?

Reply:
... or just use a JS object with `JSON.stringify({...}, null, 2)`

Reply:
I was gonna say the same. JS object and then stringify

Reply:
But that would leave the Json unformatted, no?

Reply:
No, it's formatted with `JSON.stringify({...}, null, 2)`  2 is for padding

Reply:
Yup

Reply:
Refactored.